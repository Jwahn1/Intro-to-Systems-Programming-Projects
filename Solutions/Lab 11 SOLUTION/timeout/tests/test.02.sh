#!/bin/sh

$1 4 snooze 3
