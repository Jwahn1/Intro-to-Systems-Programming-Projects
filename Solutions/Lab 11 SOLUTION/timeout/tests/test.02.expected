Good night, World!
Good morning, World!
